docs/ui/components/Snippet/SnippetHeader.tsx

```diff
@@ -40,6 +42,17 @@ export const SnippetHeader = ({
       {title}
       {showOperation && operationType ? <FileStatus type={operationType} /> : null}
     </LABEL>
+    {linkUrl && (
+      <LABEL className="flex flex-1 justify-end items-center pr-2">
+        <Button
+          size="xs"
+          theme="secondary"
+          onClick={() => window.open(linkUrl, '_blank')}
+          openInNewTab>
+          Raw
+        </Button>
+      </LABEL>
```

Comment:
We have a dedicated `SnippetAction` component which can be use to display actionable button inside of `Snippet` header:
* https://github.com/expo/expo/blob/53885f4a780dc1c65b84157bcf57641a4a7a0d0d/docs/ui/components/Snippet/SnippetAction.tsx#L9-L9